test/test-v8debugapi.ts

```diff
@@ -909,10 +947,12 @@ describe('v8debugapi', function() {
           // resulting in stringItems.length being 0.
           assert(stringItems.length === 1);
 
-          api.clear(bp);
           config.capture.maxDataSize = oldMaxData;
```

Comment:
The same concern about moving these lines inside the callback of `api.clear` as described above.